config/initializers/doorkeeper.rb

```diff
@@ -435,7 +435,7 @@
   # For example if dealing with a trusted application.
   #
   skip_authorization do |resource_owner, client|
-    true
+    true # TODO: Should this always be skipped? Probably not.
```

Comment:
...what is this doing. help.

Reply:
It's if the website should ask the user if they want to connect their Bedlam website account to the external service. (see here: https://doorkeeper.gitbook.io/guides/configuration/skip-authorization)

Since the only external application currently is the wiki, and that's also us, they don't really need to verify that they want to share their data, but if we would use Doorkeeper for something external you do want to use them for verification.

Reply:
Ah yes, given that all of this is will be replaced in the medium-term, I don't think this is worth fixing.